tests/phpunit/tests/REST_API/Embed_Controller.php

```diff
@@ -93,6 +116,9 @@ public function tearDown() {
 		$story_post_type = new Story_Post_type( $experiments, $meta_boxes );
 		$story_post_type->remove_caps_from_roles();
 
+		$this->set_permalink_structure( '' );
```

Comment:
I would add a comment to explain this.

Reply:
I don't think a cleanup in `tearDown` for something `setUp` does necessarily warrants a comment.